packages/browser/src/utils/lazyLoadIntegration.ts

```diff
@@ -68,7 +69,14 @@ export async function lazyLoadIntegration(
     script.addEventListener('error', reject);
   });
 
-  WINDOW.document.body.appendChild(script);
+  const currentScript = WINDOW.document.currentScript;
+  const parent = (currentScript && currentScript.parentElement) || WINDOW.document.body || WINDOW.document.head;
+
+  if (parent) {
+    parent.appendChild(script);
+  } else {
+    logger.error(`Could not find parent element to insert lazy-loaded ${name} script`);
```

Comment:
I opted to only log an error and not throw here because this likely is an edge case behavior which our users wouldn't test against. So I'd rather let `lazyLoadIntegration` never resolve/reject than throwing/rejecting a promise. Happy to change if reviewers have different opinions.

Reply:
I think I'd rather not do that, because users code may then pend forever, which is probably harder to debug than an error in this case? 🤔 But no strong feelings...

Reply:
I guess it comes down to how SDK consumers use the API. If they don't use it in a blocking way (i.e. let other program code depend on the integration being loaded), we should be good. But sure, they can also just `await` it and have important logic come afterwards... I'll revert to throwing an error.